lib/try.js

```diff
@@ -1,17 +1,75 @@
 var express = require('express');
 var request = require('request');
+var streamBuffers = require("stream-buffers");
 
 module.exports = function() {
   var proxy = new express.Router();
   
-  proxy.all('/', function(req, res) {
+  proxy.all('/', function( req, resp ) {
+    var requestedUrl = getRequestedUrl( req );
+    enrichResponseWithRequestHeaders( req, resp );
+    var requestBuffer = new streamBuffers.WritableStreamBuffer({
```

Comment:
Why are we using StreamBuffer here? Is the idea that streaming the request into the buffer is better than just using bodyParser/buffering the body as normal? 
Not 100% sure there'd be any difference, but could be wrong there!